src/sql/workbench/contrib/notebook/browser/outputs/notebookMarkdown.ts

```diff
@@ -138,13 +139,12 @@ export class NotebookMarkdownRenderer {
 				// mailto uris do not need additional encoding of &, otherwise it would not render properly
 				if (uri.scheme !== 'mailto') {
 					href = href.replace(/&(?!amp;)/g, '&amp;');
-				} else {
-					href = href.replace(/</g, '&lt;')
-						.replace(/>/g, '&gt;')
-						.replace(/"/g, '&quot;')
-						.replace(/'/g, '&#39;');
 				}
-				return `<a href=${href} data-href="${href}" title="${title || href}">${text}</a>`;
+				href = href.replace(/</g, '&lt;')
```

Comment:
Is this actually related to this PR? It looks like this was just a mistake from the last PR that got in

Reply:
I saw that one of the commits didn't go through via the last PR. I saw this as I was refactoring so wanted to make sure the mistake was fixed.

Reply:
It's generally better to put fixes like that in separate PRs so they don't get lost among other changes. Makes it easier to review both sets of changes then.

Reply:
Gotcha will revert this and put it in a separate PR.